pkg/ddc/jindocache/transform.go

```diff
@@ -260,6 +260,15 @@ func (e *JindoCacheEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, m
 		// only support WRITE_AROUND
 		writePolicy := "WRITE_AROUND"
 
+		if mount.Options["writePolicy"] == "WRITE_THROUGH" || mount.Options["writePolicy"] == "CACHE_ONLY" {
```

Comment:
Could you add unit test for this change?

Reply:
done